### PR TITLE
[WIP] acm-registration-scripts

### PR DESCRIPTION
**Pull request by copejon**

### CODE REVIEW

File: hack/acm-spoke.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
```
Comment (rootfs) [inline hack/acm-spoke.sh:1]: what's the file for?

File: hack/acm-hub.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  searchCollector:
+    enabled: true
+  version: 2.2.0
+EOF
+
+oc new-project "$CLUSTER_NAME" 1>/dev/null
+oc label namespace "$CLUSTER_NAME" cluster.open-cluster-management.io/managedCluster="$CLUSTER_NAME"
+oc apply -f "$WORK_DIR"/managed-cluster.yaml
+oc apply -f "$WORK_DIR"/klusterlet-addon-config.yaml
+sleep 3
```
Comment (rootfs) [inline hack/acm-hub.sh:49]: why sleep? can we do wait and watch?

**Comment by oglok** [comment]
Shouldn't this script go to the microshift-demo repo instead of here?

**Comment by fzdarsky** [comment]
Agree with @oglok, it's not a core component of MicroShift (which we try to keep minimal), but it's an option of how to use/manage it. Therefore, I'd also put it into the microshift-demos repo.

File: hack/acm-hub.sh (hunk 2 above)
Comment (copejon) [reply hack/acm-hub.sh:50]: We definitely can enhance these scripts. Given the time crunch, I wanted to keep it simple.

File: hack/acm-spoke.sh (hunk 1 above)
Comment (copejon) [reply hack/acm-spoke.sh:1]: Just to simplify the spoke registration step.

**Comment by copejon** [comment]
Fair enough, will push other there

**Pull request closed without merging**